package.json

```diff
@@ -52,10 +52,12 @@
 		"@floating-ui/dom": "^1.6.13",
 		"@libsql/client": "^0.14.0",
 		"@observablehq/plot": "^0.6.16",
+		"arctic": "^3.2.2",
 		"codemirror": "^6.0.1",
 		"d3": "^7.9.0",
+		"dompurify": "^3.2.4",
 		"drizzle-orm": "^0.39.1",
-		"isomorphic-dompurify": "^2.21.0",
```

Comment:
Due to:
https://github.com/vercel/next.js/discussions/58142#discussioncomment-10239943

Throw a 500 HTTP error on Cloudflare Worker and logs:
```
ReferenceError: window is not defined
```